src/vs/workbench/browser/quickopen.ts

```diff
@@ -247,20 +248,30 @@ export class EditorQuickOpenEntry extends QuickOpenEntry implements IEditorQuick
 	}
 
 	public run(mode: Mode, context: IEntryRunContext): boolean {
-		if (mode === Mode.OPEN) {
+		const hideWidget = mode === Mode.OPEN;
+
+		let opts = this.getOptions();
+		if (mode === Mode.OPEN_BEHIND) {
+			opts = objects.assign(opts || {}, {
+				pinned: true,
+				revealIfVisible: true,
```

Comment:
@wprater I think this option is a bit dangerous here because what it basically means is that the file will not be opened if it is visible in any other group. but I think the intent when sending files to open to the background is to open them in that group independent from other groups. I suggest to not set this flag.